slave-prod-centos7/Dockerfile

```diff
@@ -18,4 +19,8 @@ RUN curl -Lk https://password.corp.redhat.com/cacert.crt -o /etc/pki/ca-trust/so
     ln -sf /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem /etc/pki/tls/certs/ca-bundle.crt && \
     update-ca-trust && update-ca-trust enable
 
+COPY ./ssh_config /etc/ssh/ssh_config
+RUN mkdir /home/rhmap4-build
+RUN chmod 777 /home/rhmap4-build
```

Comment:
Why do we need this dir?

Reply:
Ansible plays expect it, as they were designed to run within the docker @pb82 originally developed